### PR TITLE
Add resources page.

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: resources/resources.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.. _resources:
+
+Resources
+================
+The resources provided in the ``resources`` folder are available for use under CC-BY license |CC-BY|_.
+The original authors are named either in the documents or in the list down below.
+
+Any files submitted to the documentation will be "licensed" under ``ros2_control`` as organization and stating your name, e.g., ``CC-BY My Name (ros2_control)``.
+
+Presentations
```
Comment (bmagyar) [inline resources/resources.rst:10]: shouldnt' we add yours and Andy's presentations from ROS World?

Comment (destogl) [reply resources/resources.rst:10]: Yeah, we could probably also do this.

File: resources/resources.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+    Controllers expose ROS interfaces for 3rd party solutions to robotics problems like manipulation path planning (`moveit2`) and autonomous navigation (`nav2`). The modular design makes it ideal for both research and industrial use. A robot made up of a mobile base and an arm that supports ros2_control needs no extra code, only a few controller configuration files and it is ready to
+
+  Recording: TBA
+
+  *Presenter: Bence Magyar*
+
+  Authors:
+    - Bence Magyar (FiveAI Ltd)
+    - Denis Stogl (Stogl Robotics Consulting)
```
Comment (bmagyar) [inline resources/resources.rst:28]:
```suggestion
    - Dr. Bence Magyar (FiveAI Ltd)
    - Dr. Denis Stogl (Stogl Robotics Consulting)
```

File: resources/resources.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+  Summary:
+    With the release of ros2_control and MoveIt 2, ROS 2 Foxy finally has all the “ingredients” needed to power a robot with similar features as in ROS 1. We present the driver for Universal Robot’s manipulators as a real-world example of how robots can be run using ROS 2. We show how to realize multi-interface support for position and velocity commands in the driver and how to support scaling controllers while respecting factors set on the teach pendant. Finally, we show how this real-world example influences development of ros2_control to support non-joint related inputs and outputs in its real-time control loop.
+
+  Recording: TBA
+
+  *Presenter: Denis Štogl*
+
+  Authors:
+    - Denis Štogl (PickNik Inc.)
```
Comment (bmagyar) [inline resources/resources.rst:41]:
```suggestion
    - Dr. Denis Štogl (PickNik Inc.)
```

File: resources/resources.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  Summary:
+    One of the top reasons to upgrade from ROS1 to ROS2 is better suitability for realtime tasks. We discuss the development of a new ROS2 controller to handle realtime contact tasks such as tool insertion with industrial robots. The admittance controller handles trajectories and single-waypoint streaming commands, making it compatible with MoveIt and many teleoperation frameworks. Part of the work involved ensuring kinematic limits (position/velocity/acceleration/jerk) are obeyed while limiting interaction forces with the environment. Finally, we give practical recommendations and examples of the admittance controller. A live demo will be shown at our booth.
+
+  Recording: TBA
+
+  *Presenter: Dr. Andy Zelenak*
+
+  Authors:
+    - Dr. Andy Zeleank (PickNik Inc.)
+    - Denis Štogl (PickNik Inc.)
```
Comment (bmagyar) [inline resources/resources.rst:59]:
```suggestion
    - Dr. Denis Štogl (PickNik Inc.)
```

**Review by bmagyar — COMMENTED** [review]
Let's all be doctors (yes yes you haven't had the defense yet but I can promote you!) :D

**Comment by bmagyar** [comment]
thanks!

**Pull request merged**